README.md

```diff
@@ -1,24 +1,241 @@
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> 1-3 key features of the application.
+
+- **Record Posts**
+- **Record Users**
+- **Record Comments**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Link to the deployed project.
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  cd my-folder
+  git clone https://github.com/didierganthier/ruby-group-capstone
+```
+
+### Install
+
+Install this project with:
+
+```
+  cd rails-social-media
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+  rails s
+```
+
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+  rspec spec/app_spec.rb
+```
+
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> All of the collaborators of this project.
+
+👤 **Mhamad Raad**
+
+- GitHub: [@Mhamad-Raad](https://github.com/Mhamad-Raad)
+- LinkedIn: [Mhamad Raad](https://www.linkedin.com/in/mhamad-raad)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> 1 - 3 features to add to the project.
+
+- [ ] **Share Posts**
+- [ ] **Login and Signup**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> A message to encourage readers to support the project
+
+If you like this project and want to support it, please give it a ⭐️!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Giving credit to everyone who inspired the codebase.
+
+I would like to thank:
+
+- [Microverse](https://www.microverse.org/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
```

Comment:
Great job @Mhamad-Raad 🤝

- [ ] [Optional] You may fix the broken MIT license link to make your readme file more professional.